amethyst_utils/src/scene.rs

```diff
@@ -23,22 +25,25 @@ use serde::Serialize;
 /// - `M`: `Format` to use for loading `Mesh`es from file
 #[derive(Deserialize, Serialize)]
 #[serde(default)]
-pub struct BasicScenePrefab<V, M = ObjFormat>
+pub struct BasicScenePrefab<V, R, M = ObjFormat>
```

Comment:
Can't this also have a default to avoid changing all the examples like you have done?

Reply:
No because that's really up to the user to use it. The examples don't make use of it so that's why I changed them.

Reply:
I don't quite understand your logic right now... What does it hurt to just make it `R = ()` so that current code doesn't break and you don't have to bother adding the `()` if you don't care about it?

Reply:
People might use just BasicScenePrefab<V>, not realize that the Removal component is a thing and skip on that feature. Tbh I use Removal in 100% of the prefabs I have made up to date.

Reply:
They would miss it anyway. They copy from the examples and just ignore that `()` and then they'll find out later once they do need it. It's essentially the same thing, just with more initial confusion if it always has to be there. Besides, not everyone will actually use that feature but might do it some other way.

Reply:
okay will change